rust_src/src/windows.rs

```diff
@@ -672,6 +672,19 @@ pub fn window_live_p(object: Option<LispWindowRef>) -> bool {
     object.map_or(false, |m| m.is_live())
 }
 
+/// Return new pixel size of window WINDOW.
+/// WINDOW must be a valid window and defaults to the selected one.
+/// The new pixel size of WINDOW is the value set by the last call of
+/// `set-window-new-pixel' for WINDOW.  If it is valid, it will be shortly
+/// installed as WINDOW's pixel height (see `window-pixel-height') or pixel
+/// width (see `window-pixel-width').
+
+#[lisp_fn]
+pub fn window_new_pixel(window: LispWindowValidOrSelected) -> EmacsInt {
+    let win: LispWindowRef = window.into();
+    EmacsInt::from(win.new_pixel)
```

Comment:
We usually right this as `win.new_pixel.into()`.
```suggestion
    win.new_pixel.into()
```